sdk/cognitiveservices/service.projects

```diff
@@ -0,0 +1,6 @@
+<Project>
+  <ItemGroup>
+  <ProjectReference Remove="$(MSBuildThisFileDirectory)Azure.InkRecognizer.WPF.Stroke\**\*.csproj" />
```

Comment:
Please fix the indention here and remember to update these when you rename the folders.

Reply:
done. Thanks